### PR TITLE
fix: Add support for the LongEmbed benchmark (#393)

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
(adds missing points)

* add the longembed benchmark

* add longembed bench & make lint

* add meta data and model scores

* add all metadata and passkey&needle scores

* remove prints

* replace context length with test_256, test_512, ...

### CODE REVIEW

File: docs/mmteb/points.md
Comment (KennethEnevoldsen) [inline docs/mmteb/points.md:1]: @dwzhu-pku and @isaac-chung added points here for #393

Comment (KennethEnevoldsen) [reply docs/mmteb/points.md:1]: Yea, me too (damn!) - I realized this yesterday - feel free to do a PR duplicating the review scores

Comment (KennethEnevoldsen) [reply docs/mmteb/points.md:1]: did it here: #435

Comment (isaac-chung) [reply docs/mmteb/points.md:1]: Thanks! I did mine via https://github.com/embeddings-benchmark/mteb/pull/425 so will remove the changes to mine.

**Pull request merged**